stream_alert/alert_processor/outputs.py

```diff
@@ -14,17 +14,24 @@
 limitations under the License.
 """
 from abc import abstractmethod
-import cgi
 from collections import OrderedDict
 from datetime import datetime
+import cgi
```

Comment:
Please move this to it's previous location, per our conversation with @austinbyers  a few weeks ago.

The convention we now follow post his pylint refactor was that imports will be alphabetical by top level package (regardless of import/from grouping). This change is now inconsistent with all other import orders throughout the code base.

Reply:
I personally like grouping by `from` and `import` but ok